flags.go

```diff
@@ -435,6 +435,7 @@ type flagStorage struct {
 	// Monitoring & Logging
 	StackdriverExportInterval  time.Duration
 	OtelCollectorAddress       string
+	PrometheusPort             string
```

Comment:
I think we want to remove all mentions of `PrometheusPort` from this file.

Reply:
Actually, a recent commit https://github.com/GoogleCloudPlatform/gcsfuse/commit/d0868157afd71d08f2f02f11c802ca3bd97f71bb#diff-84fa0fc3b79e063575d5e65751f76d79c1f961ef42facf47fd225d65e0ff5cfb shows that config file and mount flag are added at the same time, modified this PR accordingly.